guide/sections/part2/global-services.adoc

```diff
@@ -202,9 +204,52 @@ wis2-gdc provides the functionality required for the Global Discovery Catalogue,
 
 wis2-gdc is managed as a free and open source project. Source code, issue tracking and discussions are hosted openly on GitHub: https://github.com/wmo-im/wis2-gdc.
 
-==== 2.7.6 Global Monitor
+==== 2.7.6 Global Replay
 
 ===== 2.7.6.1 Technical considerations
+
+* The Global Replay provides Global Services and data consumers with a mechanism to search, query and subscribe to notification messages and monitoring events of interest.
+* The Global Replay implements the OGC API – Features – Part 1: Core standardfootnote:[https://docs.ogc.org/is/17-069r4/17-069r4.html], adhering to the following conformance classes and their dependencies:
+** Core
+** GeoJSON
+* The Global Replay implements the OGC API – Processes - Part 1: Core standardfootnote:[https://docs.ogc.org/is/18-062r2/18-062r2.html], adhering to the following conformance classes and their dependencies:
+** Core
+** OGC Process Description
+** JSON
+* A Global Replay shall subscribe to the topics `+origin/a/wis2/#+`, `+cache/a/wis2/#+` and `+monitor/a/wis2/#+`.
+* The Global Replay will make notification messages available via the collection identifier `wis2-notification-messages`.
+* The Global Replay will make monitoring events available via the collection identifier `wis2-monitoring-events`.
+* The Global Replay will provide a process to allow for users to subscribe to messages with various criteria (spatial and / or temporal extent, topic selection) via MQTT.
+* A single Global Replay instance is sufficient for WIS2.
+* Multiple Global Replay instances may be deployed for resilience.
+* Global Replay instances operate independently of each other; each Global Replay instance will hold notification messages and monitoring events according to the required retention period.  Global Replays do not need to synchronise between themselves.
+* A Global Replay is populated with notification messages and monitoring events from a Global Broker instance.
+* A Global Replay should connect and subscribe to more than one Global Broker instance to ensure that no messages or events are lost in the event of a Global Broker failure. A Global Replay instance will discard duplicate messages as needed.
+* A Global Replay can validate notification messages against the WIS Notification Message (WNM).  Valid WIS Notification Messages will be ingested into the index.  Invalid or malformed notification messages will be discarded.
```

Comment:
Using **index** here sounds like a technical solution.

A Global Replay can validate notification messages against the standard format (see Manual on WIS, Volume II – Appendix E. WIS2 Notification Message). Valid messages will be stored. Invalid or malformed messages will be discarded.